spec/features/contact_centre_agent_creates_appointments_spec.rb

```diff
@@ -0,0 +1,69 @@
+# rubocop:disable Metrics/MethodLength
+require 'rails_helper'
+
+RSpec.feature 'Contact centre agent creates appointments' do
```

Comment:
I wonder if we can shorten _contact centre agent_ to _agent_ without losing any meaning in the domain? cc @benbarnett 

If we agree on this we should shorten it everywhere it's currently referred to as _contact centre agent_.